### ISSUE TITLE
Cannot resolve module 'switchery'

### ISSUE THREAD

**Issue by offdev** [mohithg/react-switchery#4]
When trying to transpile using webpack, I get following error:
```
ERROR in ./~/react-switchery/build/index.js
Module not found: Error: Cannot resolve module 'switchery' in /project/laravel/resources/assets/node_modules/react-switchery/build
 @ ./~/react-switchery/build/index.js 17:17-37
```

**Comment by mohithg** [comment]
@offdev Thanks for pointing the issue. I fixed it in `1.1.3`, remove `react-switchery` from your node_modules and do `npm install`.

**Comment by mohithg** [comment]
Closing issue since its fixed
